docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
+Once again in the image above, note the file `GigasecondTests.swift` inside the `GigasecondTests` folder. This file is unnecessary and can be deleted or moved somewhere else. Removing it is helpful because we are about to replace it with a new file of almost the same name.
 
-## Running Tests
+#### Import The Exercism Tests Source File
+![008-testsimportexample](https://cloud.githubusercontent.com/assets/5385825/20957618/c7800d4c-bc05-11e6-937f-de2f3743c1b6.png)
 
-Test will run through the Xcode GUI using the XCTest framework.
+Right-Click (or Control-Click) on the `GigasecondTests` folder and choose Add Files to "Gigasecond" from the contextual menu to open a file import dialog box. Navigate to and select the unit tests file in the exercise folder. In this example the file is named `GigasecondTest.swift`. Different exercises may vary the name format somewhat.
+
+Add the line `@testable import Gigasecond`. (If the Xcode project name contains a hyphen, replace it with an underscore. For example, with Word-count, enter `@testable import Word_count`). The red icons indicate failed tests or missing code objects, which is normal at this stage. Once imported correctly it should resemble the image above.
+
+#### Make A New Source File For Your Code
+
+Right-Click (or Control-Click) on the `Gigasecond` group folder and choose `New File`. Select macOS and Swift from the template chooser. The filename is up to you, but using the name of the exercise itself is a good starting point. It can be changed later. By convention Swift classes and structs begin with a capital letter, so their associated source files should as well. **Note this is the file you will submit to the Exercism web site for review.**
+
+Housekeeping is finshed and Xcode is ready! Here is how the file inspector should look now:
+
+![009-finallayoutexample](https://cloud.githubusercontent.com/assets/5385825/20957619/c7832748-bc05-11e6-9626-2f8a01fdcf22.png)
+
+The playground is useful for drafting code snippets and playing with ideas.
+
+`Gigasecond.swift` is the code file where you'll solve the exercise.
+
+`GigasecondTest.swift` contains the tests provided by Exercism.
+
+### Understanding How Tests Work
+
+Each project has two targets, the application target, and the Unit Tests target. The source files for these targets appear in different groups (the yellow folders) in the Xcode file inspector. 
+
+As part of the configuration in the previous section, you added a source file inside each group. 
+
+The application source file is where you'll write the code needed to make the tests pass.
+
+The tests source file was provided as part of the exercise. If in the course of solving the exercise you consider editing test source, stop, because you are missing something. The exercise can be solved with code in the application source file alone.
```

Comment:
I would add here a bit more of information, as I think a lot of people get stuck here. Something like...:
_The exercise can be solved with code in the application source file alone. Once you've written a correct code in your application source file, those errors you see now in the tests source file should disappear. But that doesn't mean that the exercise is finished! Then you need to run the tests as we explain in the next step. _